idowuadeleke/BANKA
# chore(refactor codebase ): refactored codebase based on feedback

#### What does this PR do?
Implements feedback gotten from LFA to the codebase

#### Description of Task to be completed?
- change status code from 422 to 400 and from 201 to 200
- Onclick banka image should redirect back to index page
- Replace account type on the index page to Role
- Console log listening port
- The sidebar should be raised on mobile view
- Remove commented code
- Add a line at end of page

#### How should this be manually tested?
User Interface can be tested y visiting the link below
https://idowuadeleke.github.io/BANKA/

Running app with npm run start to check console logged port

Using postman to test status code

#### What are the relevant pivotal tracker stories?
#165336755

#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/40268693/56147627-90304780-5fa0-11e9-8f5e-f0de8bb275ea.png)
![image](https://user-images.githubusercontent.com/40268693/56147701-a8a06200-5fa0-11e9-85e6-86685800db4c.png)
